gulp/html.js

```diff
@@ -274,7 +280,8 @@ module.exports = (config, cb) => {
               metatags: metatags.generateTags(metatagsData),
               breadcrumb: breadcrumb.sort((a, b) => {
                 return a.url.length - b.url.length
-              })
+              }),
+              fileHistory: `https://github.com/Access4all/adg/blame/main/pages/${relPath}`
```

Comment:
Create Github link. The blame view seemed to be the most useful one to see both the file content and the history.

Reply:
We switched to the commit history.